docs/datasets_and_tables/transitdatabase.md

```diff
@@ -2,14 +2,52 @@
 
 ### Assessments
 
+| **Name**<br>*Key(s)*| **Description** |
+| :------------- |  :-------------- |
+| `gtfs checks` | Each record represents a check that can be performed on either a(n):<ul><li>`organization` (who manages a transit service)</li><li>`gtfs dataset` as a whole, or </li><li>a `gtfs service data` record within that `gtfs dataset`. Each check has a score type of either:<ul><li>`Boolean`: yes or no</li><li>`Nominal`: with defined scores for specific criteria levels as defined in the `Scoring Criteria` field, or</li><li>`Continuous`: as defined by the `Scoring Criteria` column and the `Max Score` field.</li></ul>Further, scores for each check can be sourced by one of <ul><li>`human`: manual work</li><li>`auto`: programmable</li><li>`gtfs-trained-human`: a huma with necessary technical training, or </li><li>`combo`: of these.</li></ul>These sources are indicated in their current, near-future, and goal states in the columns `Source`, `Source: medium-term`, and `Source: goal`. |
+| `gtfs-service check data` | `WIP` Each record is a specific assessment of a single `gtfs checks` for a single `gtfs service data` record at a specific point in time. |
+| `gtfs-dataset check data` | `WIP` Each record is a specific assessment of a single `gtfs checks` for a single `gtfs dataset` record at a specific point in time. |
+| `provider check data` | `WIP` Each record is a specific assessment of a single `gtfs checks` for a single `organizations` which *manages* one or more `services` at a specific point in time. |
```

Comment:
I recommend dropping `gtfs` prefix from datasets to be agnostic to dataset type (could include GBFS in future for example):

```suggestion
| `provider check data` | `WIP` Each record is a specific assessment of a single `dataset checks` for a single `organizations` which *manages* one or more `services` at a specific point in time. |
```